src/components/fx/hover.js

```diff
@@ -1089,75 +1221,92 @@ function cleanPoint(d, hovermode) {
     return d;
 }
 
-function createSpikelines(hoverData, opts) {
-    var hovermode = opts.hovermode;
+function createSpikelines(closestPoints, opts) {
     var container = opts.container;
-    var c0 = hoverData[0];
-    var xa = c0.xa;
-    var ya = c0.ya;
-    var showX = xa.showspikes;
-    var showY = ya.showspikes;
+    var fullLayout = opts.fullLayout;
+    var evt = opts.event;
+    var xa,
+        ya;
+
+    var showY = closestPoints.hLinePoint ? true : false;
```

Comment:
maybe:

```js
var showY = !!closestPoints.hLinePoint;

// or
var showY = Boolean(closestPoints.hLinePoint);
```

would suffice?

Reply:
Thanks, changed it to `var showY = !!closestPoints.hLinePoint;`